HealthMonitoring.AcceptanceTests/Xunit/TestInitialization.cs

```diff
@@ -1,30 +1,52 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Configuration;
 using System.Data;
 using System.Linq;
 using System.Net;
-using System.Reflection;
 using System.Threading;
 using HealthMonitoring.AcceptanceTests.Helpers;
+using HealthMonitoring.AcceptanceTests.Xunit;
+using LightBDD.Core.Configuration;
+using LightBDD.Framework.Reporting.Configuration;
+using LightBDD.Framework.Reporting.Formatters;
+using LightBDD.XUnit2;
 using MySql.Data.MySqlClient;
 using Newtonsoft.Json;
 using RestSharp;
-using Xunit;
-using Xunit.Abstractions;
-using Xunit.Sdk;
 
-[assembly: TestFramework("HealthMonitoring.AcceptanceTests.Xunit.TestFramework", "HealthMonitoring.AcceptanceTests")]
+[assembly: ConfiguredLightBddScope]
+
 namespace HealthMonitoring.AcceptanceTests.Xunit
 {
+    class ConfiguredLightBddScopeAttribute : LightBddScopeAttribute
+    {
+        protected override void OnSetUp()
+        {
+            TestInitalization.Initialize();
+        }
+
+        protected override void OnTearDown()
+        {
+            TestInitalization.Terminate();
+        }
+
+        protected override void OnConfigure(LightBddConfiguration configuration)
+        {
+            configuration.ReportWritersConfiguration()
+                .Clear()
+                .AddFileWriter<XmlReportFormatter>(@"~\Reports\FeaturesSummary.xml")
+                .AddFileWriter<HtmlReportFormatter>(@"~\Reports\FeaturesSummary.html")
+                .AddFileWriter<PlainTextReportFormatter>(@"~\Reports\FeaturesSummary.txt");
+        }
+    }
```

Comment:
This class allows to configure LightBDD2 but also to start services before tests and stop after test execution